### DISCUSSION TITLE
Distance snap not snapping to radius

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#21325] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/78441708/202891890-2aecfc80-38a8-4d5e-ae88-78f355840817.png)
it appears that while the UI that shows the distance snap is accurate and not tied to SV, the actual distance snap still is, so when the distance snap is after a slider with SV =/= 1 then it is inaccurate to the UI.

**Comment by Axilotl17** [comment]
for clarification: placing a bunch of hitcircles with distance snap works fine, and the visuals are correct (i think) but when you introduce SV there is a discrepancy between the visuals and the actual distance snap.

**Comment by peppy** [comment, marked as answer]
This was already fixed in a recent build (https://github.com/ppy/osu/pull/20850)... but seems to have regressed. Will investigate.

**Reply by peppy** [reply]
Tracking via #21347..
